rosette/src/Meta.cc

```diff
@@ -136,11 +136,30 @@ pOb StdMeta::get(pOb client, pOb key, pCtxt) {
      * This unfolding of the valWrt function pays big dividends during
      * method lookup.
      */
+
     switch (GET_GENERIC_TYPE(loc)) {
     case LT_LexVariable:
         if (GET_LEXVAR_IND(loc)) {
             container = ((Actor*)client)->extension;
+
+// TODO: This prevents a crash caused by a malformed expander. See Jira ticket ROS-304.
+//
+//       This is a blatant BANDAID and a HACK, and needs to be fixed in the compiler
+//       to not generate malformed objects.
+//       The comparison agains 0x100 is because, while the pointer is "NULL", it really
+//       isn't zero because of the additional "Tag" data that in this implementation gets
+//       overlayed on pOb pointers.
```

Comment:
Is there somewhere in the code that these tag bits are documented?  If not, sketch it out in the proper place.  In either case, add a pointer to that description here.

Reply:
There isn't a central location for the TAG bits implementation.  It is sprinkled in various places in the code in the form of nested preprocessor macros. In another branch I'm attempting to refactor this so that the TAGs don't live inside the pointers, but in a separate variable within the Ob class.

At the moment, I'm pulling this into the Ob class, but it may eventually land within its own class that is closely related to the Ob class.

The hope here was that we could get this fix out, and then work on the whole TAG , Heap and Object management handling as a separate effort. At that point, this fix will clean up and not be so ugly.

Reply:
OK, then say something like, "The low six bits are used as tag bits.  See, for example, the FOO macro in Bar.cc and the BAZ macro in Gunk.cc".

Reply:
I've done this.  Can you accept the PR now?